pkg/randchar/generator_test.go

```diff
@@ -175,6 +175,16 @@ func TestAdd(t *testing.T) {
 			b:        "",
 			expected: "",
 		},
+		"different": {
+			a:        "ab",
+			b:        "bc",
+			expected: "abc",
+		},
+		"different with overlap": {
```

Comment:
I think the names of these cases are swapped. `"ab", "bc"` have overlap, `"ab", "c"` have not.